.harness/github_push.yaml

```diff
@@ -0,0 +1,26 @@
+inputSet:
+  name: github_push
+  tags: {}
+  identifier: github_push
+  orgIdentifier: default
+  projectIdentifier: IPA_Release
+  pipeline:
+    identifier: aiohttp_responses
+    template:
+      templateInputs:
+        properties:
+          ci:
+            codebase:
+              build:
+                type: branch
+                spec:
+                  branch: <+trigger.branch>
+        stages:
+          - parallel:
+              - stage:
+                  identifier: snyk_tests
+                  type: CI
+                  variables:
+                    - name: RUN_SNYK
+                      type: String
+                      value: "FALSE"
```

Comment:
in hindsights, we are not actually running snyk and not sure if we want to

Reply:
still not sure how we feel about this lib in general. this was just something we threw together because we couldn't find a good alternative at the time. definitely needs some love

Reply:
enabling snyk should be fine though. we can just be lax in dependency versions here.